### PR TITLE
Fixes Create/Delete rest api calls

### PR DESCRIPTION
**Pull request by brendankowitz**

### CODE REVIEW

**Comment by RandomEngy** [comment]
Where are the real changes? They seem to be buried under some this. prefix refactoring

**Comment by brendankowitz** [comment]
Yeah I updated settings, the `this.` stuff is getting in the way :(
I'll see if I can separate somehow...

Basically its changes like this to support "No Content":
https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/35/files#diff-3a9d9161447aed85813a38f1fc21d762R1132

**Comment by davidkwoods** [comment]
Try comparing between commit 1 and 2.  That looks promising.

---
In reply to: [302169154](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/35#issuecomment-302169154) [](ancestors = 302169154)

File: Microsoft.HealthVault.RestApi/Generated/Models/ActionPlan.cs [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
-
 namespace Microsoft.HealthVault.RestApi.Generated.Models
 {
+    using Microsoft.HealthVault;
+    using Microsoft.HealthVault.RestApi;
+    using Microsoft.HealthVault.RestApi.Generated;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
```
Comment (davidkwoods) [inline Microsoft.HealthVault.RestApi/Generated/Models/ActionPlan.cs]: Any particular reason we moved the usings to inside the namespace?

**Comment by RandomEngy** [comment]
:shipit:

File: Microsoft.HealthVault.RestApi/Generated/Models/ActionPlan.cs (hunk 1 above)
Comment (brendankowitz) [inline Microsoft.HealthVault.RestApi/Generated/Models/ActionPlan.cs]: I think I moved the namespaces with resharper on initial checkin which fixed the them. This is the raw output from autorest (https://github.com/Azure/autorest/blob/master/src/generator/AutoRest.CSharp/Templates/Rest/Client/ServiceClientTemplate.cshtml#L13)

Comment (davidkwoods) [inline Microsoft.HealthVault.RestApi/Generated/Models/ActionPlan.cs]: Fair enough.

**Pull request merged**